### DISCUSSION TITLE
Is this right? To override default options of a plugin, I need to set `opts`

### DISCUSSION THREAD

**Discussion by mertnuhoglu** [LazyVim/LazyVim#626] [Q&A] [answered]
The documentation of a plugin suggests the following code to customize the settings:

```lua
  require('github-theme').setup({
    options = {
      transparent = false,
  ...
```

In LazyVim, instead I will create a file in `lua/plugins/github_theme.lua` with the following code:

```lua
  return {
    { "projekt0n/github-nvim-theme" },
    {
      "LazyVim/LazyVim",
      opts = {
        options = {
          transparent = false,
    ...
```

Is this right?

**Comment by folke** [comment]
no, you want to set the options for the githubnvim-theme plugin, so add opts there.
Please, please, please just read the docs. it's all in there and has tons of examples.

**Comment by mertnuhoglu** [comment]
Thank you for the reply and sorry for the inconvenience. I really read the documents and checked the examples. But I am confused between `opts` and `options`. 

This one is correct, right?

```lua
return {
  {
    "projekt0n/github-nvim-theme",
    opts = {
      transparent = false,
  ...
```

This one is wrong, right?

```lua
return {
  {
    "projekt0n/github-nvim-theme",
    opts = {
      options = {
        transparent = false,
  ...
```

**Reply by dpetka2001** [reply, marked as answer]
You could either try the first one which is correct or you could even try the `config` option of `lazy.nvim` and setup your plugin like you're used to already. For example:
```lua
return {
  {
    "projekt0n/github-nvim-theme",
    config = function()
      require('github-theme').setup({
		options = {
		  transparent = false,
 		 ...
  },
}
```

**Reply by dpetka2001** [reply]
It would benefit you if you could also read the [docs](https://github.com/folke/lazy.nvim) of `lazy.nvim` which describe all the options you can use for setting up your specs with Lazyvim.
